ppr-ui/src/interfaces/store-interfaces/account-model-interface.ts

```diff
@@ -0,0 +1,9 @@
+export interface AccountModelIF {
+  currentAccount: {
+    id: number
+  }
+  currentAccountMembership: any
+  currentUser: Object
+  pendingApprovalCount: number
+  userSettings: Array<Object>
```

Comment:
Are these properties used?

Reply:
No, they are not used, but thought of keeping them in the IF. I could remote to make it cleaner.

Reply:
Totally, the interface is a welcome addition, i'm just stumped as to the purpose of the segregated state 🤷